get-started/code-experiments/src/prepare.py

```diff
@@ -0,0 +1,108 @@
+import struct
+import numpy as np
+import gzip
+import os
+from util import load_params
+
+
+def mnist_images_idx_to_array(images_filename):
+    images_f = gzip.open(images_filename, mode="rb")
```

Comment:
as we discussed - let's simplify this, let's not use `gz` and this format - let's use regular images